content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -151,7 +153,10 @@ stages:
 by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
 the possible following fields:
 
-- `cmd` (always present): Executable command defined in this stage
+- `cmd` (always present): One or more commands executed by the stage (may
+  contain either a single value, or a list). Commands are executed sequentially
+  until all are finished or until one of them fails (see
+  [`dvc repro`](/doc/command-reference/repro) for details).
```

Comment:
Note for self:

- [ ] The details of how commands are executed should go somewhere in the `repro` command reference.

Reply:
Done in 56252e34